workflow/scripts/get_pathway_barplot.R

```diff
@@ -0,0 +1,20 @@
+log <- file(snakemake@log[[1]], open = "wt")
```

Comment:
Script can be removed in favor of the datavzd report.

Reply:
Noted.